packages/ndk/analysis_options.yaml

```diff
@@ -16,4 +16,3 @@ linter:
   # `// ignore_for_file: name_of_lint` syntax on the line or in the file
   # producing the lint.
   rules:
-    - public_member_api_docs
```

Comment:
before we had not linter for api_docs. So I am not so sure if removing is the best way forward. Do you know if there is a warning level below info?
Ideally we have something that still screams: hey you forgot api docs here

Reply:
Maybe we can increase the severity of other linter rules, and leave this one as info (there is no level below info AFAIK) and hopefully we'll add docs to everything in time.

Reply:
I feel this linter rule goes way beyond what makes sense to document. It's literally showing warnings on hundreds of methods. Maybe it's better to start writing on the most important methods. Also general documentation would be super helpful. 

As my 2c feedback, feel free to ignore, I find it a bit hard to understand the project: first it's a *lot* of code, then when I browse the code I can't find many patterns, also I need to go study clean architecture. Maybe a general documentation site would add a lot of value for newbs like me